libs/langchain-community/package.json

```diff
@@ -239,6 +240,7 @@
     "@google-cloud/storage": "^6.10.1 || ^7.7.0",
     "@gradientai/nodejs-sdk": "^1.2.0",
     "@huggingface/inference": "^2.6.4",
+    "@langchain/langgraph": "^0.0.25",
```

Comment:
Let's set this dep to `~0.0.25` so that people can automatically resolve to versions up to `0.1.0`

Reply:
👍